pkg/reconciler/brokercell/resources/deployments.go

```diff
@@ -127,6 +127,12 @@ func MakeRetryDeployment(args RetryArgs) *appsv1.Deployment {
 
 // deploymentTemplate creates a template for data plane deployments.
 func deploymentTemplate(args Args, containers []corev1.Container) *appsv1.Deployment {
+	annotation := map[string]string{
+		"sidecar.istio.io/inject": strconv.FormatBool(args.AllowIstioSidecar),
+	}
+	if args.RolloutRestartTime != "" {
+		annotation["events.cloud.google.com/RestartedAt"] = args.RolloutRestartTime
```

Comment:
I think this annotation key should be a constant.

Reply:
I think "RestartedAt" could be misleading - the deployment could be restarted for other reasons, right? Basically I don't want this annotation to be considered the authority of last restart time.

Can we call this something like "RestartRequestedAt"?